test/built-ins/TypedArray/out-of-bounds-get-and-set.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-isvalidintegerindex
+description: >
+  Getting and setting in-bounds and out-of-bounds indices on TypedArrays backed
+  by resizable buffers.
+includes: [resizableArrayBufferUtils.js]
+features: [resizable-arraybuffer]
+---*/
+
+for (let ctor of ctors) {
+  if (ctor.BYTES_PER_ELEMENT != 1) {
+    continue;
+  }
```

Comment:
Fine with me if we decide to leave this as is, but is there a reason why we skip non-1-byte typed arrays, instead of multiplying the resizes by `ctor.BYTES_PER_ELEMENT` as in many of the other tests? (I feel like there is probably a good reason, I'm just missing it.)

Reply:
I switched to the style of the other tests and the tests were only failing for BigInt TAs, which was fixed by using `MayNeedBigInt` as in other RAB tests.